2020/day-5-binary-boarding/index.ts

```diff
@@ -0,0 +1,71 @@
+const firstHalf = (arr: any[]) => arr.slice(0, arr.length / 2)
```

Comment:
```suggestion
const firstHalf = (arr: string[]) => arr.slice(0, arr.length / 2)
```

Reply:
I think I decided to use `any` here since this is a generic function that doesn't care about the contents of the array